satellite/payments/tokens.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package payments
+
+import (
+	"context"
+	"math/big"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+)
+
+// StorjTokens defines all payments STORJ token related functionality.
+type StorjTokens interface {
+	// Deposit creates deposit transaction for specified amount.
+	Deposit(ctx context.Context, amount big.Float) (*Transaction, error)
+}
+
+// TransactionStatus defines allowed statuses
+// for deposit transactions.
+type TransactionStatus string
```

Comment:
I'm wondering, for consistency reasons, if this type shouldn't be an `int` like https://github.com/storj/storj/blob/acd51e44b36ffee261884c512cf123622f71c339/satellite/payments/coinpayments/transactions.go#L41

Reply:
`coinpayments` is an implementation detail, it could possibly be different provider for token payment, with different status type. My idea is that it might be easier for caller to work with string value of statuses, but yeah it might be no difference actually.